Faraday_Proxy_Tools/FaradayIO/deviceconfig.py

```diff
@@ -285,7 +285,7 @@ def update_gps(self, gps_boot_bitmask, latitude_str, latitude_dir_str, longitude
 
         else:
             print "ERROR: GPS string(s) too long OR NMEA DMM formatting incorrect"
-            print "ERROR: Altitude must be %g-%g" % (MIN_ALTITUDE_METERS, MAX_ALTITUDE_METERS)
+            print "ERROR: Altitude must be %g-%g" % (self.MIN_ALTITUDE_METERS, self.MAX_ALTITUDE_METERS)
```

Comment:
Hey @rtucker thanks for the pull request! How about we use Python .format() instead of % for string variable and formatting? This would correspond to the rest of Faraday Software + is the preferred method as of Python 2.6. It's also the preferred method for Python 3.0 if we move towards that down the line.